app/build.gradle.kts

```diff
@@ -20,15 +23,18 @@ plugins {
     alias(libs.plugins.kapt)
     alias(libs.plugins.ksp)
     alias(libs.plugins.hilt)
-}
+
+    }
+
+
 
 android {
-    compileSdk = libs.versions.compileSdk.get().toInt()
 
+    compileSdkPreview = "UpsideDownCake"
     defaultConfig {
         applicationId = "com.example.android.architecture.blueprints.main"
-        minSdk = libs.versions.minSdk.get().toInt()
-        targetSdk = libs.versions.targetSdk.get().toInt()
+        minSdk = 33
```

Comment:
This should still be the ones referenced from the libs file.

Reply:
> This should still be the ones referenced from the libs file.
 and what is that file named?